lsp/client/vscode/src/sso/model.ts

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// For the Proof of concept, this file's code was copied from the AWS Toolkit for VS Code repo
```

Comment:
mentioning the commit sha (or better, a github url with the commit sha) helps a lot

Reply:
Good idea, I'll add these